docroot/modules/custom/sitenow_articles/sitenow_articles.module

```diff
@@ -98,6 +98,17 @@ function sitenow_articles_preprocess_field(&$variables) {
       $variables['element']["#title"] = 'Written by';
       $variables["label"] = 'Written by';
     }
+    // Loop through all author fields in the byline and add credentials
+    // (if available) to their names.
+    foreach ($variables['items'] as $key => $item) {
+      $author_entity = $item['content']['#options']['entity'];
+      if ($author_entity->hasField('field_person_credential') && !$author_entity->get('field_person_credential')
+        ->isEmpty()) {
+        $credentials = $author_entity->field_person_credential->value;
+        $authorTitle = $variables['items'][$key]["content"]["#title"];
+        $variables['items'][$key]["content"]["#title"] = $authorTitle . ", " . $credentials;
```

Comment:
Can we switch these double quotes to single?

Reply:
Also, can we switch `$authorTitle` => `$author_title`?

Reply:
I guess I thought that snake case was the coding standard for Drupal variables, but it appears either is okay: https://www.drupal.org/docs/develop/standards/coding-standards#s-functions-and-variables. In that case, I'm requesting it just to be consistent with how we name variables everywhere else.

Reply:
yeah! no prob on both items. I swear I'd switched camel case -> snake case at some point, but I may have accidentally undid it

Reply:
Also, with PHP8's [nullsafe operator](https://www.php.net/releases/8.0/en.php#nullsafe-operator), I think we might be able to rewrite this as:
```php
$credentials = $author_entity
  ?->field_person_credential
  ?->value;
if (!is_null($credentials)) {
  $variables['items'][$key]['content']['#title'] .= ', ' . $credentials;
}
```

Reply:
Dang, that's some hot-off-the-presses syntax: VS Code hates it, but it does seem to work. Going to do some further testing to ensure it does.

Reply:
It does seem to work! Removed the clunky if-statement and replaced with the nullsafe operator. Fun.